### ISSUE TITLE
✅ 🔗 🐞 Ajax Pagination + Dynamic Background Image: Background image not showing on next page when using Ajax Pagination with Loop Grid (Duplicate of #22368) - [ED-12415]

### ISSUE THREAD

**Issue by Daveden2** [elementor/elementor#24330]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When using a dynamic background image such as the post's featured image or custom field image, the background image does not load when you switch to the next page using AJAX load type.

It is probably related to this [other bug](https://github.com/elementor/elementor/issues/22368)

### Steps to reproduce

1. Add a loop grid to an Elementor Page
2. Create a new template and apply the featured image as the background image of the container.
3. Choose any pagination type such as Numbers or Prev/Next
4. Set the Load Type to AJAX
5. Test on frontend

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.54 (Win64) PHP/8.0.30 mod_fcgid/2.3.10-dev
	MySQL version: mariadb.org binary distribution v10.10.2-MariaDB
	PHP Version: 8.0.30
	PHP Memory Limit: 512M
	PHP Max Input Vars: 2500
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.1
	Site URL: http://ele-staging.local
	Home URL: http://ele-staging.local
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.9.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.2.3
		Author: WP Engine

	Elementor
		Version: 3.17.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.17.1
		Author: Elementor.com

	FakerPress
		Version: 0.6.1
		Author: Gustavo Bordoni

	HappyFiles Pro
		Version: 1.8.3
		Author: Codeer

== Elements Usage ==
	
	header : 1
		container : 1
		image : 1
		social-icons : 1
	wp-page : 3
		container : 4
		heading : 1
		image : 1
		image-gallery : 1
		loop-grid : 3
	loop-item : 2
		container : 4
		heading : 1
		post-info : 1
		theme-post-featured-image : 1
		theme-post-title : 1


== Settings ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved asset loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active
	Hello Theme Header &amp; Footer: Active
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Active
	Build with AI: Active
	Landing pages: Active
	Nested Elements: Active
	Lazy Load Background Images: Active
	Optimize Image Loading: Active
	Pages Panel: Inactive by default
	Global Style Guide: Active
	Page Transitions: Active
	Notes: Active
	Form Submissions: Active
	Scroll Snap: Active
	Menu: Active
	Taxonomy Filter: Active


== Log ==
	


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by watels** [comment]
Same issue for me

**Comment by YuvrajKhavad** [comment]
Same issue we have.

**Comment by nicholaszein** [comment]
Hey everyone!

I know I'm little late, but I'm just passing to let you know that...

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.19.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
